string binsInputSentence
string binsClientQid
---
string binsOutput
